slack_notifier.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "curb"
+
+class SlackNotifier
+  def initialize(slack_webhook)
+    @slack_webhook = slack_webhook
+  end
+
+  def new_issue_not_published
+    send_message("*Fíha, dnes na ÚVO nevyšlo nové vydanie vestníka?*")
+  end
+
+  def matching_announcements(page, announcements)
+    send_header_message(page, announcements)
+
+    announcements.each do |a|
+      send_message("<#{a[:link][:href]}|#{a[:link][:text]}>: *#{a[:customer]}* #{a[:description]}")
```

Comment:
Ok. Počkaj už s komentármi na ďalšiu iteráciu keď to prečistím. Začínam sa tu v tom strácať.